examples/solverdummies/cpp/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(SolverdummyCPP LANGUAGES CXX DESCRIPTION "preCICE CPP solver dummy")
 find_package(precice REQUIRED CONFIG)
 
 add_executable(solverdummy solverdummy.cpp)
-set_target_properties(solverdummy PROPERTIES CXX_STANDARD 17)
+set_target_properties(solverdummy PROPERTIES CXX_STANDARD 11)
```

Comment:
Our solverdummies could be a good test case for our API compatibility with C++11. Why upgrading here?

Reply:
Have a look at the colour :stuck_out_tongue_winking_eye:

Reply:
Time to go home